src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -196,31 +210,44 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
     // Create lifecycle for the bucket with a list of rules
     val lifecycle = new Lifecycle().setRule(List(lifecycleRule).asJava)
 
+    // The Leo service account
     val leoServiceAccountEntityString = s"user-${dataprocConfig.serviceAccount.string}"
-    val userServiceAccountEntityString = s"user-${userServiceAccount.value}"
-
-    //Add the Leo SA and the user's pet SA to the ACL list for the bucket
-    val bucketAcls = List(
-      new BucketAccessControl().setEntity(leoServiceAccountEntityString).setRole("OWNER"),
-      new BucketAccessControl().setEntity(userServiceAccountEntityString).setRole("READER")
-    )
-
-    //Bucket ACL != the ACL given to individual objects inside the bucket
-    val defObjectAcls = List(
-      new ObjectAccessControl().setEntity(leoServiceAccountEntityString).setRole("OWNER"),
-      new ObjectAccessControl().setEntity(userServiceAccountEntityString).setRole("READER")
-    )
 
-    // Create the bucket object
-    val bucket = new Bucket()
-      .setName(bucketName.name)
-      .setLifecycle(lifecycle)
-      .setAcl(bucketAcls.asJava)
-      .setDefaultObjectAcl(defObjectAcls.asJava)
-
-    val bucketInserter = storage.buckets().insert(googleProject.string, bucket)
+    val clusterServiceAccountEntityStringFuture: Future[String] = if (dataprocConfig.createClusterAsPetServiceAccount) {
```

Comment:
Not for now, but expect a refactoring to a more provider-y mechanism sometime. See also: integrations w/sam.